ui/components/app/flask/snap-install-warning/index.scss

```diff
@@ -0,0 +1,43 @@
+.snap-install-warning {
+  .checkbox-label {
+    @include H7;
+
+    display: flex;
+    align-items: flex-start;
+    gap: 0 16px;
+  }
+
+  &__footer {
+    display: flex;
+    flex-flow: row;
+    justify-content: center;
+    flex: 0 0 auto;
+    width: 100%;
+    max-height: 40px;
+  }
+
+  &__footer-button {
+    margin-right: 16px;
+
+    &:last-of-type {
+      margin-right: 0;
+    }
+  }
+
+  .popover-header {
```

Comment:
What are these styling changes for? Typically we try to scope all style changes to the namespace for the component, `snap-install-warning` in this case. Adding rules for other classes can introduce subtle dependencies between componments that can be broken accidentally fairly easily. For example if you rename a class within the `Popover` component, change layout or rearrange classes/components, etc. It makes the styles part of the public API of the component, which we don't want.

Reply:
Hmmm, I wanted to add in these rules to this usage of the `Popover` component in `SnapInstallWarning`, I thought throwing the rules under `.snap-install-warning` to avoid any conflicts. Not sure how I can have the style changes I need without doing this or either customizing the `Popover` component to accept css rules.

Reply:
Ideally we'd use the same style for all popovers so they have a consistent look/feel. What motivated these changes?

Reply:
The flask designs themselves I suppose, the padding it already had didn't seem sufficient.

Reply:
Could you leave those changes out, and make them to the base Popover class instead? I don't think there is anything special about this popover that warrants different padding than anywhere else in the extension.

Reply:
If I remove the `Popover` css from `SnapInstallWarning`'s css, add a wrapper div ('snap-install-warning__content`) around what get's displayed in the `popover-content` div and add some padding (because the popover-content div doesn't provide that), the result is below. Do you think that's fine, visually?

<img width="373" alt="Screen Shot 2022-01-20 at 10 58 56 AM" src="https://user-images.githubusercontent.com/41640681/150374955-766803c6-eeef-4972-8870-977d886d8a1a.png">

Reply:
I guess this eliminates the need for me to change the `Popover` component and makes the content portion look decent, although I'd argue with 24px padding on the sides, things look better. This way I also don't have to add the `Popover` css in `SnapInstallWarning` at all.

Reply:
>Do you think that's fine, visually?

I mean... it doesn't look _terrible_ to me. Certainly passable enough that it's a step forward.

I think I agree that the 24px padding looks better. Just we should use the same popover styles throughout the app to maintain a consistent look and feel. So we should pursue that change in the popover component itself rather than for this specific instance.

Reply:
Okay, I pushed up the change. I'll customize the `Popover` component further in a separate PR and then loopback and adjust the content padding to 24px in this component.